src/lsf.jl

```diff
@@ -63,6 +58,11 @@ function lsf_bpeek(manager::LSFManager, jobid, iarray)
         end
     end
 
+    if !worker_started
+        close(stream)
+        throw(LSFException(bytestr))
```

Comment:
Looks like `bytestr` is defined inside the `for` loop body, so this may throw `UndefVarError`?

Reply:
Ooops, good catch! Should be fixed now.